.travis.rosinstall

```diff
@@ -1,3 +0,0 @@
-- git:
-    uri: https://github.com/ros-industrial/abb.git
-    local-name: ros-industrial/abb
```

Comment:
Ok. That would work, as `abb` is now a source repo in the `.rosinstall` file. Reason a separate `.travis.rosinstall` was added is because it is/was the recommended way to get additional source dependencies in an `industrial_ci` Travis session while making it explicit that these are not normally needed to be cloned as sources in the workspace.